_includes/nav/page-item.html

```diff
@@ -1,8 +1,8 @@
 {% include nav/current.html page=include.page %}
 <a href="{{ include.page.url | relative_url }}"{%
-  if _current -%} class="usa-current"{% endif %}>{{
-    include.page.title | smartify
-  }}</a>
+  if _current -%} class="usa-current"{% endif %}>
+  <span>{{ include.page.title | smartify }}</span>
+</a>
```

Comment:
@donjo do you know how I can add the span to the header nav but not the sidenav?